ansible/roles/nginx/vars/cas_ssl.yml

```diff
@@ -103,6 +109,11 @@ nginx_sites:
          - zone_name: "submissions"
            burst: 100
            nodelay: True
+     - name: "/hq/admin/session_details"
+       balancer: "{{ deploy_env }}_cas_commcare"
+       proxy_redirect: "http://{{ CAS_SITE_HOST }} https://{{ CAS_SITE_HOST }}"
+       proxy_next_upstream_tries: 1
+       proxy_read_timeout: 900s
```

Comment:
strange that this should need a separate zone since the traffic is from an local IP.

Reply:
Because we have multiple rate limiting configurations the IP white list will need to be added to all of them. This was hitting our generic '/' rate limiting (by server_name) so it needed to be pulled out https://github.com/dimagi/commcarehq-ansible/blob/3664dd35f550937df8e230295022321c06c8054f/ansible/roles/nginx/vars/cas_ssl.yml#L117-L124